### PR TITLE
Update color variables in SelectMenu

### PR DESCRIPTION
**Pull request by bscofield**
Updating SelectMenu and its children

### CODE REVIEW

File: src/SelectMenu/SelectMenuTabPanel.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {MenuContext} from './SelectMenuContext'
 import SelectMenuList from './SelectMenuList'
 
 const TabPanelBase = styled.div<SystemCommonProps & SxProp>`
-  border-top: ${get('borderWidths.1')} solid ${get('colors.border.gray')};
+  border-top: ${get('borderWidths.1')} solid ${get('colors.border.secondary')};
```
Comment (colebemis) [inline src/SelectMenu/SelectMenuTabPanel.tsx:10]:
```suggestion
  border-top: ${get('borderWidths.1')} solid ${get('colors.selectMenu.borderSecondary')};
```

File: src/SelectMenu/SelectMenuLoadingAnimation.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const pulseKeyframes = keyframes`
 const Animation = styled.div<SystemCommonProps>`
   padding: ${get('space.6')} ${get('space.4')};
   text-align: center;
-  background-color: ${get('colors.white')};
+  background-color: ${get('colors.bg.canvas')};
```
Comment (colebemis) [inline src/SelectMenu/SelectMenuLoadingAnimation.tsx:23]:
```suggestion
  background-color: ${get('colors.bg.overlay')};
```

**Review by colebemis — APPROVED** [review]
Looks great! Just left a few suggestions. After those are resolved and you've added a changeset this PR should be good to go 🚢

**Comment by bscofield** [comment]
@colebemis 👍  changes merged, changeset added, ready for merge when you have time

File: src/SelectMenu/SelectMenuFilter.tsx [hunk 3]
```diff
@@ -9,8 +9,8 @@ import {MenuContext} from './SelectMenuContext'
 const StyledForm = styled.form<SystemCommonProps & SxProp>`
   padding: ${get('space.3')};
   margin: 0;
-  border-top: ${get('borderWidths.1')} solid ${get('colors.border.gray')};
-  background-color: ${get('colors.white')};
+  border-top: ${get('borderWidths.1')} solid ${get('colors.selectMenu.borderSecondary')};
```
Comment (colebemis) [inline src/SelectMenu/SelectMenuFilter.tsx:12]:
```suggestion
  border-bottom: ${get('borderWidths.1')} solid ${get('colors.selectMenu.borderSecondary')};
```

**Pull request merged**